base/src/main/java/io/fabric8/launcher/base/identity/RSAPublicKeyConverter.java

```diff
@@ -0,0 +1,56 @@
+package io.fabric8.launcher.base.identity;
+
+import java.math.BigInteger;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.RSAPublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.RSAPublicKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+
+import org.apache.commons.codec.binary.Base64;
```

Comment:
How about using [`java.util.Base64`](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html) instead of adding a dependency to commons-codec?

Reply:
Funny that IDEA didn't show me that as auto-import... :confused: 

`commons-codec` though is already part of the project - but seems not to be used.

86215721 fixes that